summarization/extractor_output_preprocessor.py

```diff
@@ -112,6 +117,20 @@ def _split_content(self):
             elif content.content_type != ContentType.UNKNOWN:
                 self.embedded_content_list.append(content)
 
+    def _strip_numbering_from_title_text(self):
+        for title_text in self.title_text_content_list:
+            title_text.text_string \
+                = self._strip_numbering_prefix_from_text(
+                    title_text.text_string)
+
+    def _strip_numbering_prefix_from_text(self, text):
+        numbered_prefix_list = re.findall('^[0-9]+[:,.,)]*', text)
+        if numbered_prefix_list == []:
+            return text
+        numbered_prefix = numbered_prefix_list[0]
+
+        return text[len(numbered_prefix):]
```

Comment:
nit: Is this used anywhere else ?- it's fine to just merge (would prefer it) with the above method.